control-plane/config/eventing-kafka-broker/200-webhook/500-webhook.yaml

```diff
@@ -109,7 +109,13 @@ spec:
                 - name: k-kubelet-probe
                   value: "webhook"
           livenessProbe:
-            <<: *probe
```

Comment:
we do use these in eventing core as well

Reply:
> we do use these in eventing core as well

do you mean custom tag? that's true, only the source is different. in eventing-core, it comes from [eventing-core.yaml](https://github.com/knative/eventing/blob/main/config/core/deployments/webhook.yaml#L113) and the mentioned tag comes from https://github.com/knative-extensions/eventing-kafka-broker/blob/main/control-plane/config/eventing-kafka-broker/200-webhook/500-webhook.yaml#L103